apis/v1alpha1/nginxproxy_types.go

```diff
@@ -27,12 +27,33 @@ type NginxProxyList struct {
 	Items           []NginxProxy `json:"items"`
 }
 
+// IPFamilyType specifies the IP family to use for all servers.
+//
+// +kubebuilder:validation:Enum=ipv4;ipv6;both
+type IPFamilyType string
```

Comment:
@salonichf5 FYI: looks like you pushed your IPv6 changes to this PR...

Reply:
oh yeah. weird, fixing it